pantheon/src/test/java/tech/pegasys/pantheon/cli/PantheonCommandTest.java

```diff
@@ -528,6 +529,35 @@ public void genesisPathDisabledUnderDocker() {
     assertThat(commandOutput.toString()).isEmpty();
   }
 
+  @Test
+  public void enableP2pOptionDefaultTrue() {
+    parseCommand("--p2p-enabled");
+
+    verify(mockRunnerBuilder.p2pEnabled(eq(true))).build();
+
+    assertThat(commandOutput.toString()).isEmpty();
+    assertThat(commandErrorOutput.toString()).isEmpty();
+  }
+
+  @Test
+  public void enableP2pOptionMustBeUsed() {
```

Comment:
The name of this test has nothing to do with what is being tested, has it?

Reply:
I'll rework that a bit...